arma3-windows/entrypoint.ps1

```diff
@@ -0,0 +1,213 @@
+function Ensure-Installation {
+    if (!(Test-Path -Path "C:/arma3server/steamcmd/steamcmd.exe" -PathType leaf)) {
+        New-Item -Path "C:/arma3server" -Name "steamcmd" -ItemType "directory"
+
+        $(cd "steamcmd")
+
+        $(curl.exe -L -o steamcmd.zip https://steamcdn-a.akamaihd.net/client/installer/steamcmd.zip)
+
+        Expand-Archive ./steamcmd.zip .
+
+        Start-Process "C:/arma3server/steamcmd/steamcmd.exe" -ArgumentList "+login anonymous +quit" -Wait -NoNewWindow
+
+        Write-Output "SteamCMD Setup complete"
+    }
+
+    if (!(Test-Path -Path "C:/arma3server/X3DAudio1_7.dll" -PathType leaf)) {
+        Copy-Item "C:/temp/X3DAudio1_7.dll" "C:/arma3server/X3DAudio1_7.dll"
+        Write-Output "Copied 3DAudio"
+    }
+    if (!(Test-Path -Path "C:/arma3server/XAPOFX1_5.dll" -PathType leaf)) {
+        Copy-Item "C:/temp/XAPOFX1_5.dll" "C:/arma3server/XAPOFX1_5.dll"
+        Write-Output "Copied APOFX"
+    }
+}
+
+function error($1) {
+    Write-Error "[ips-error] $1"
+}
+
+function Get-LatestRpt()
+{
+    $latestRpt = $null
+
+    try 
+    {
+        $latestRpt = Get-ChildItem "logfiles" -Filter "*.rpt" `
+                    | Sort-Object LastWriteTime -Descending `
+                    | Select-Object -First 1
+    }
+    catch { }
+
+    return $latestRpt
+}
+
+function Compare-SteamCredentials {

```

Comment:
Maybe Check-SteamCredentials is a more appropriate name?

Reply:
Sure, In the beginning I tried to use the Accepted Verbs for Powershell Functions but those are quite limited
https://docs.microsoft.com/de-de/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.2